README.md

```diff
@@ -56,11 +43,11 @@ If you find any issues on the project you would like to report, please create an
-| @faststore/sdk | Exports basic logic hooks for creating your custom ecommerce                                | [![npm version](https://badge.fury.io/js/%40faststore%2Fsdk.svg)](https://badge.fury.io/js/%40faststore%2Fsdk) [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/sdk)](https://bundlephobia.com/package/@faststore/sdk) [![tree shaking](https://badgen.net/bundlephobia/tree-shaking/@faststore/sdk)](https://bundlephobia.com/package/@faststore/sdk) [![tree shaking](https://badgen.net/bundlephobia/dependency-count/@faststore/sdk)](https://bundlephobia.com/package/@faststore/sdk) |
-| @faststore/api | Connect your site to your ecommerce provider                                                |                                                                                                                                                                                             [![npm version](https://badge.fury.io/js/%40faststore%2Fapi.svg)](https://bundlephobia.com/package/@faststore/api)                                                                                                                                                                                              |
+| @faststore/sdk | Exports basic logic hooks for creating your custom ecommerce                                                       | [![npm version](https://badge.fury.io/js/%40faststore%2Fsdk.svg)](https://badge.fury.io/js/%40faststore%2Fsdk) [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/sdk)](https://bundlephobia.com/package/@faststore/sdk) [![tree shaking](https://badgen.net/bundlephobia/tree-shaking/@faststore/sdk)](https://bundlephobia.com/package/@faststore/sdk) [![tree shaking](https://badgen.net/bundlephobia/dependency-count/@faststore/sdk)](https://bundlephobia.com/package/@faststore/sdk) |
+| @faststore/api | Connect your site to your ecommerce provider                                                                       |                                                                                                                                                                                             [![npm version](https://badge.fury.io/js/%40faststore%2Fapi.svg)](https://bundlephobia.com/package/@faststore/api)                                                                                                                                                                                              |
```

Comment:
Maybe we should add `faststore/core` here too.

Reply:
Agreed :)